packages/fast-components-react-msft/app/examples.ts

```diff
@@ -1,5 +1,9 @@
 import ButtonExamples from "../src/button/examples.data";
 export {ButtonExamples};
 
+import MediaExamples from "../src/media/examples.data";
+
```

Comment:
can you remove this new line? Also, we had to update the tslint rules to check import/export statements to include spaces...so, the rule is that in import/export we should have spaces, but not elsewhere...A rebase should fix the other examples here and you can align with those.